src/script/sign.cpp

```diff
@@ -499,3 +499,19 @@ FlatSigningProvider Merge(const FlatSigningProvider& a, const FlatSigningProvide
     ret.origins.insert(b.origins.begin(), b.origins.end());
     return ret;
 }
+
+bool IsSegWitOutput(const SigningProvider& provider, const CScript& script)
+{
+    std::vector<valtype> solutions;
+    auto whichtype = Solver(script, solutions);
+    if (whichtype == TX_WITNESS_V0_SCRIPTHASH || whichtype == TX_WITNESS_V0_KEYHASH || whichtype == TX_WITNESS_UNKNOWN) return true;
+    if (whichtype == TX_SCRIPTHASH) {
```

Comment:
Commit cb085728f64a76fec48dee4e6432c199aa69a62a

Isn't this new code?

Reply:
Yes?

Reply:
That's fine, just that after reading commit message I thought this was move only commit.

Reply:
Ah, good point. I'll clarify in the commit message.

Reply:
Done.